solutions/project/spoke-project-unclass/setters.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  # Billing Account ID to be associated with this project
+  project-billing-id: "AAAAAA-BBBBBB-CCCCCC"
+  # GCP folder to use as parent to this project, lowercase K8S resource name
+  project-parent-folder: project-parent-folder
+  # Naming Convention for project-id : <tenant-code><environment-code>m<data-classification>-<project-owner>-<user defined string>
+  # Max 30 characters
+  project-id: xxdmu-admin1-projectname
+  nane1-paz-snet: 10.1.1.0/24
```

Comment:
please have a description/comment for every setter

Reply:
done